docs/api/python_client.md

```diff
@@ -1,38 +1,28 @@
 # 🐍 Python Client API Reference
 
 ::: llmengine.Completion
-    selection:
-        members:
-            - create
-            - acreate
+selection:
+members: - create - acreate
```

Comment:
What does moving them all on one line do?

Reply:
this comes from vscode formatter